posthog/queries/trends/sql.py

```diff
@@ -98,7 +98,6 @@
 """
 
 TOP_ELEMENTS_ARRAY_OF_KEY_SQL = """
-SELECT groupArray(value) FROM (
```

Comment:
Running through `groupArray()` removes the rows where this field is `None`. We however do want to know if `None` is part of the returned fields or not.